pkg/console/util.go

```diff
@@ -97,6 +98,10 @@ func customizeConfig() {
 		cfg.Config.Runcmd = append(cfg.Config.Runcmd, fmt.Sprintf(`keys=$(curl -sfL --connect-timeout 30 %q) && echo "$keys">>%s`, cfg.Config.SSHKeyURL, authorizedFile))
 	}
 
+	cfg.Config.K3OS.Labels = map[string]string{
+		"harvester.cattle.io/version": version.Version,
```

Comment:
Why do we need the version label on nodes? We will need to consider the upgrade for this